plan/plan.go

```diff
@@ -224,27 +230,19 @@ type Module struct {
 type Account struct {
 	ComponentCommon `yaml:",inline"`
 
-	AccountBackends map[string]Backend      `yaml:"account_backends"` // Reference accounts for remote state
-	AllAccounts     map[string]*json.Number `yaml:"all_accounts"`
-	AccountName     string                  `yaml:"account_name"`
-	Global          *Component
+	Account string
+	Global  *Component
```

Comment:
do these need yaml tags? (since the ones removed had them)

Reply:
Yeah problem, will make plan output easier to read.